comptests/prov/test_prov_k8s_cortx.py

```diff
@@ -431,3 +451,93 @@ def test_32654(self):
                                            data_disk_per_cvg=config['data_disk_per_cvg'],
                                            master_node_list=self.master_node_list,
                                            worker_node_list=self.master_node_list)
+
+    @pytest.mark.lc
+    @pytest.mark.comp_prov
+    @pytest.mark.tags("TEST-41569")
+    def test_41569(self):
+        """
+        S3-IO Operations.
+        """
+        LOGGER.info("Test Started.")
+        LOGGER.info("Step 1: Get Access and Secret Key.")
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_GET_ACCESS_KEY, read_lines=True)
+        access_key = "".join(resp).replace("\\n", "\n")
+        LOGGER.info(access_key)
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_GET_SECRET_KEY, read_lines=True)
+        secret_key = "".join(resp).replace("\\n", "\n")
+        LOGGER.info(secret_key)
+        LOGGER.info("Step 2: Fetch Interface IP, HTTP and HTTPS Ports.")
+        resp = system_utils.execute_cmd(cmd=commands.CMD_GET_IP_IFACE.format
+                                        (self.deploy_cfg['iface']))
+        LOGGER.info(resp)
+        eth1_ip = resp[1].strip("'\\n'b'")
+        LOGGER.info(eth1_ip)
```

Comment:
Please log  information of the  variable 'eth1_ip'.
for example: LOGGER.info(f"IP of ethernet 1: {eth1_ip}").

Also do the same for all the info logging.